components/style/values/computed/font.rs

```diff
@@ -192,3 +192,38 @@ impl ToComputedValue for specified::MozScriptMinSize {
         specified::MozScriptMinSize(ToComputedValue::from_computed_value(other))
     }
 }
+
+/// Make type alias with `i8`
```

Comment:
Let's change this comment to say: `/// The computed value of the -moz-script-level property.`.